include/llbuild/BuildSystem/BuildSystem.h

```diff
@@ -112,6 +112,11 @@ class BuildSystemDelegate {
                      const Token& at,
                      const Twine& message) = 0;
 
+  /// Called to provide informational output of the build system.
+  ///
+  /// \param message The informational message.
+  virtual void output(const Twine& message) = 0;
```

Comment:
I don't think we want this at the BuildSystem level, do we? Shouldn't we have it on a per-command basis, similar to `commandProcessHadOutput`?